sql/tsql/TSqlLexer.g4

```diff
@@ -893,6 +888,13 @@ XACT_ABORT:                            'XACT_ABORT';
 VARCHAR:                               'VARCHAR';
 NVARCHAR:                              'NVARCHAR';
 
+//Combinations that cannot be used as IDs
+BINARY_BASE64:                         'BINARY BASE64';
+DISK_DRIVE:                            [A-Z][:];
+GO_BATCH:                              [\r\n]* [\t ]* 'GO'[\t\n\r ];
```

Comment:
It looks weird. Probably it's better to check if `GO` starts with new line using semantic predicates. Also, it may affect performance.

Reply:
I agree. Of course this was there before, I only moved it, but I will check the semantic predicate to clean that up. Thanks!

Reply:
Actually, I did simplify the logic around go and got rid of this token.